cli/docker/app/factory.go

```diff
@@ -8,10 +8,12 @@ import (
 	"github.com/docker/libcompose/project"
 )
 
-type ProjectFactory struct {
+// DockerProjectFactory is a struct that hold the app.ProjectFactory implementation.
+type DockerProjectFactory struct {
```

Comment:
Oh.. I didn't see it like that. I'll revert this one, agre with you, it makes it redundant.